addons/zylann.hterrain/shaders/multisplat16.shader

```diff
@@ -52,13 +55,16 @@ vec4 pack_normal(vec3 n, float a) {
 // Blends weights according to the bump of detail textures,
 // so for example it allows to have sand fill the gaps between pebbles
 vec4 get_depth_blended_weights(vec4 splat, vec4 bumps) {
-	float dh = 0.2;
-
+	float dh = height_boost;
+	bumps.r = 1.0 - pow(1.0 - bumps.r, 0.5);
+	bumps.g = pow(bumps.g, 2.0);
```

Comment:
`pow` again, prefer squaring by multiplying by itself. Are GLSL compilers smart enough to replace the call with that? Still no idea why you do this though